### PR TITLE
Update base images device types page

### PR DESCRIPTION
**Pull request by nghiant2710**
Connects-to: #1902 
Dynamically generate content from api v6/device_type endpoint

Change-type: patch
Signed-off-by: Trong Nghia Nguyen <nghiant2710@gmail.com>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: templates/_machines-architectures.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  (function () {
+    window.jQuery
+      .ajax({
+        url: "https://api.balena-cloud.com/v6/device_type?$expand=is_of__cpu_architecture($select=slug)"
+      })
+      .done(function (deviceTypes) {
+
+        var table;
+
+        table = _(deviceTypes.d)
```
Comment (thgreasi) [inline templates/_machines-architectures.html:12]: Any reason we don't declare it inline?
```suggestion
        var table = _(deviceTypes.d)
```

File: templates/_machines-architectures.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          .sortBy([
+            function (deviceType) {
+              return (deviceType.name);
+            }
+          ])
```
Comment (thgreasi) [inline templates/_machines-architectures.html:17]: Let's drop this after adding an $orderby in the OData query above.

File: templates/_machines-architectures.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<script src="//cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" type="text/javascript"></script>
+<script type="text/javascript">
+  (function () {
+    window.jQuery
+      .ajax({
+        url: "https://api.balena-cloud.com/v6/device_type?$expand=is_of__cpu_architecture($select=slug)"
```
Comment (thgreasi) [inline templates/_machines-architectures.html:6]:
```suggestion
        url: "https://api.balena-cloud.com/v6/device_type?$select=name,slug&$expand=is_of__cpu_architecture($select=slug)&$filter=is_default_for__application/any(idfa:((idfa/is_host%20eq%20true)%20and%20(idfa/is_archived%20eq%20false))%20and%20not(contains(idfa/app_name,%27esr%27)))&$orderby=name%20asc"
```

**Review by thgreasi — APPROVED** [review]

**Comment by 20k-ultra** [comment]
@balena-ci rebase

**Pull request merged**